### PR TITLE
[6.5.x] JBPM-6904 - service tasks tab missing on project with spaces and proc…

### PR DESCRIPTION
**Pull request by tsurdilo**
…ess with spaces

### CODE REVIEW

**Comment by tsurdilo** [comment]
@hasys here is the fix for RHBPMS-5129. Tested it with eap 7 locally. Please have a look. Thanks.

**Comment by tsurdilo** [comment]
This fix is for 6.5.x branch only. Fix is already on master and 7.5.x

**Comment by hasys** [comment]
Jenkins execute full downstream build

**Comment by LuboTerifaj** [comment]
Hi, I will test it but I am not sure if I am able to finish it today. Please don't merge it yet. Thanks!

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnitVFSTest.java [hunk 1]
```diff
@@ -241,23 +242,23 @@ private void verifyWidsPngsAndGifs(List<String> wids, List<String> pngs, List<St
             Collection<Asset> storedWids = repository.listAssetsRecursively("/", new FilterByExtension("wid"));
             assertEquals(wids.size(), storedWids.size());
             for (String wid : wids) {
-                repository.assetExists(wid);
+                repository.assetExists(wid.replaceAll("\\s","%20"));
```
Comment (LuboTerifaj) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnitVFSTest.java]: Spacing again. See also 253 and 261 line. It is not wrong, but from my point of view, it is better to keep the same formatting style.

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/web/server/TaskFormsServletSpacesTest.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
```
Comment (LuboTerifaj) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/web/server/TaskFormsServletSpacesTest.java]: Wrong year again.

**Comment by tsurdilo** [comment]
@LuboTerifaj amended. thanks for your review!

**Comment by tsurdilo** [comment]
@LuboTerifaj please approve if ok now. thanks.

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnitVFSTest.java [hunk 3]
```diff
@@ -241,23 +252,23 @@ private void verifyWidsPngsAndGifs(List<String> wids, List<String> pngs, List<St
...
+                repository.assetExists(wid.replaceAll("\\s", "%20"));
             }
         }
 
         if(pngs != null) {
             Collection<Asset> storedPngs = repository.listAssetsRecursively("/", new FilterByExtension("png"));
             assertEquals(pngs.size(), storedPngs.size());
             for (String png : pngs) {
-                repository.assetExists(png);
+                repository.assetExists(png.replaceAll("\\s","%20"));
```
Comment (LuboTerifaj) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnitVFSTest.java]: Missing space has not been added.

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnitVFSTest.java [hunk 4]
```diff
@@ -241,23 +252,23 @@ private void verifyWidsPngsAndGifs(List<String> wids, List<String> pngs, List<St
...
             }
         }
 
         if(gifs != null) {
             Collection<Asset> storedGifs = repository.listAssetsRecursively("/", new FilterByExtension("gif"));
             assertEquals(gifs.size(), storedGifs.size());
             for (String gif : gifs) {
-                repository.assetExists(gif);
+                repository.assetExists(gif.replaceAll("\\s","%20"));
```
Comment (LuboTerifaj) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnitVFSTest.java]: Missing space has not been added.

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/web/server/TaskFormsServletTest.java [hunk 5]
```diff
@@ -25,35 +33,57 @@
...
+import org.uberfire.backend.vfs.PathFactory;
+import org.uberfire.backend.vfs.VFSService;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+import static org.hamcrest.CoreMatchers.anyOf;
+import static org.hamcrest.CoreMatchers.is;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TaskFormsServletTest  extends RepositoryBaseTest {
```
Comment (LuboTerifaj) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/web/server/TaskFormsServletTest.java]: Redundant space, but I am OK with that.

**Comment by tsurdilo** [comment]
@LuboTerifaj amended. thanks again

**Pull request merged**